data/assets/examples/animation.asset

```diff
@@ -30,7 +30,7 @@ local animationLoop = {
       sun.LightSource
     },
     PerformShading = true,
-    DisableFaceCulling = true
+    EnableFaceCulling = true
```

Comment:
These are all true per default now, so could be removed

Also, is there a reason for why we disabled face culling before but always enable it now? (curious)

Reply:
Because I don't think there was any good reason for us to disable it before (except for the Orion nebula which is a one-sided plane).